src/Popover/Popover.js

```diff
@@ -38,37 +38,37 @@ const Popover = React.createClass({
     animation: React.PropTypes.func,
 
     /**
-     * If true, the popover will hide when the anchor scrolls off the screen
+     * If true, the popover will hide when the anchor scrolls off the screen.
      */
     autoCloseWhenOffScreen: React.PropTypes.bool,
 
     /**
-     * If true, the popover (potentially) ignores targetOrigin
-     * and anchorOrigin to make itself fit on screen,
+     * If true, the popover (potentially) ignores `targetOrigin`
+     * and `anchorOrigin` to make itself fit on screen,
      * which is useful for mobile devices.
      */
     canAutoPosition: React.PropTypes.bool,
 
     /**
-     * Use this property to render your component inside the `Popover`.
+     * Use this property to render your component inside the `Popover` element.
```

Comment:
Could you have a look and see if there's better wording used in other components for `children`?